shared/ui-components/src/types.ts

```diff
@@ -15,4 +15,5 @@ export interface Listing {
   developer: string
   credit_history: string
   rental_history: string
+  units: any
```

Comment:
I'd love to define this type somewhere/how and not have it be a complete "any"

Reply:
Yeah I'm getting lazy about the typing on JSON-backed objects since I'm not really sure what the units format is going to be.